orchestra/utils/task_lifecycle.py

```diff
@@ -488,6 +493,8 @@ def tasks_assigned_to_worker(worker):
                     status=TaskAssignment.Status.PROCESSING,
                     task__id=task_assignment.task.id,
                     assignment_counter__lt=task_assignment.assignment_counter)
+                .exclude(
```

Comment:
the exclude here would characterize paused projects as awaiting review. don't you want it for the query on line 477, and then to make `paused_task_assignments` not filter on `status=TaskAssignment.Status.PROCESSING,`?

Reply:
I decided not to remove PROCESSING. If a project is paused, it might be confusing if a worker who has several tasks on that project to see that their "future" tasks are paused.